ui/src/modules/events/store/services/event-overview-store.facade.ts

```diff
@@ -45,10 +46,11 @@ export class EventOverviewStoreFacade extends BaseStore<EventOverviewStore.State
         )
     }
 
-    setGraphDepth(graphDepth: number): void {
+    setGraphDepth(graphDepth: number, overviewType: OverviewTypeEnum = OverviewTypeEnum.LINEAGE_OVERVIEW): void {
```

Comment:
Judging from the method signature, it seems to violate a single responsibility principle.
Consider usage examples:
```typescript
// USE-CASE: Setting a graph depth
// PROBLEM: Unwanted side effect - resetting an unrelated property `overviewType`
store.setGraphDepth(5) 

// USE-CASE: Setting a graph depth
// PROBLEM: The signature doesn't look like a setter method.
// What does overview type have to do with my graph depth?
store.setGraphDepth(5, OverviewTypeEnum.LINEAGE_OVERVIEW)

// USE-CASE: Setting an overview type
// PROBLEM: Calling a wrong method and having to provide a graph depth
store.setGraphDepth(???, OverviewTypeEnum.LINEAGE_OVERVIEW)
```

Reply:
No, you do not miss anything. That's what I meant.